src/TenantSecurityException.ts

```diff
@@ -1,8 +1,8 @@
 /**
- * 0 - 199   TspServiceException
- * 200 - 299 KmsException
- * 300 - 399 SecurityEventException
- * 900 - 999 TscException
+ ** 0 - 199   TspServiceException
+ ** 200 - 299 KmsException
+ ** 300 - 399 SecurityEventException
+ ** 900 - 999 TscException
```

Comment:
It probably would've been more clear if I put a space between the *s. It was so that the markdown (which shows on hover in my editor) makes it a bulleted list. As it was before, it was just a single run-on line like 
```
0 - 199 TspService Exception 200 - 299 KmsException 300 - 399 SecurityEventException 900 - 999 TscException
``` 
and was hard to follow